CHANGELOG.md

```diff
@@ -3,6 +3,22 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 
+## [6.5.2] Preview Release
+### Added
+- Updated the samples to be usable with Eclipse directly, and updated the driver version used by the samples to 6.4.0.jre9 [#679](https://github.com/Microsoft/mssql-jdbc/pull/679)
+- Updated Gradle script for building JDBC Driver [#689](https://github.com/Microsoft/mssql-jdbc/pull/689)
+
+### Fixed Issues
+- Fixed an issue where ResultSetMetaData instances created by a ResultSet that has been closed were not persisting [#685](https://github.com/Microsoft/mssql-jdbc/pull/685)
+- Fixed an issue with PreparedStatement.setBigDecimal when no scale is passed [#684](https://github.com/Microsoft/mssql-jdbc/pull/684)
+- Fixed an issue with Statement.setQueryTimeout not throwing an exception when no network is available [#674](https://github.com/Microsoft/mssql-jdbc/pull/674)
+- Fixed an issue with SQLServerDatabaseMetaData having performance degradation in 6.4.0 [#677](https://github.com/Microsoft/mssql-jdbc/pull/677)
```

Comment:
Please change description to PR title "**Improved performance degradation while maintaining JDBC compliance with results from sp_fkeys**". Move this one to top of the Fixed Issues List.